app/views/charge.njk

```diff
@@ -145,16 +145,24 @@
                       </p>
                     {% endif %}
                   </label>
+                  
+                  {% if allowMoto and motoMaskCardNumberInput %} 
+                    {% set cardNumberInputType = 'password' %}  
+                  {% else %}
+                    {% set cardNumberInputType = 'text' %}   
+                  {% endif %}
+
                   <input id="card-no"
-                    type="text"
-                    inputmode="numeric"
-                    pattern="[0-9]*"
-                    name="cardNo"
-                    maxlength="26"
-                    class="govuk-input govuk-!-width-three-quarters"
-                    autocomplete="cc-number"
-                    value="{{cardNo}}"
-                  />
+                      type="{{cardNumberInputType}}"
+                      inputmode="numeric"
+                      pattern="[0-9]*"
+                      name="cardNo"
+                      maxlength="26"
+                      class="govuk-input govuk-!-width-three-quarters"
+                      autocomplete="cc-number"
```

Comment:
In the words of the HTML Standard, `autocomplete="cc-number"` is “inappropriate for a control” with `type="password"`, so we should probably just not include the `autocomplete` attribute or use `autocomplete="off"`.

The same will apply to `autocomplete="cc-csc"` for the card security code field.

Reply:
I thought `autocomplete="off"` would be the best option - as it is more explicit.  Even though the browser may override this setting.

I have updated the `security code` field and the Cypress test.